app/django-girls/page.jsx

```diff
@@ -60,7 +60,7 @@ export default function DjangoGirls() {
                 <Link
                   className="text-white bg-pyconug-lightBlue px-12 py-4 rounded-md w-50 mt-10"
                   target="_blank"
-                  href=""
+                  href="https://forms.gle/K5nELeYzpfMR8yDf6"
```

Comment:
@Amanya-Brian what if we had these links in the `utils/constants.js` directory and reference them by a variable name...

Reply:
Thanks for the reminder. On it.